examples/complete/variables.tf

```diff
@@ -45,3 +45,9 @@ variable "existing_sm_instance_region" {
   description = "The region of the existing Secrets Manager instance. Only required if value is passed into var.existing_sm_instance_guid"
   default     = null
 }
+
+variable "custom_metadata" {
```

Comment:
Don't expose new variables to examples, we don't want example to be consumable solutions. Just hard code it in the main.tf of the example and leave a code comment saying here is how to add custom metadata